docs/installation.md

```diff
@@ -169,11 +191,6 @@ Clone the git repository:
 ```bash
 git clone https://github.com/freqtrade/freqtrade.git
 cd freqtrade
-```
-
-Optionally checkout the master branch to get the latest stable release:
```

Comment:
I would've left this (maybe as comment within the code-block).

changing to master is not a requirement, as also develop is kept in a stable state.

Reply:
on the one hand it is (contains latest editions, ...) - but on the other it's pretty stable nonetheless - as it's the base for the release (almost always without change other than the version number).